src/components/Modals/UploadDocumentModal.jsx

```diff
@@ -69,11 +64,8 @@ const UploadDocumentModal = ({ showModal, setShowModal }) => {
     e.preventDefault();
 
     if (!docType) {
```

Comment:
I haven't tested, but I don't think this is needed either...  I believe the `DocumentSelection` is `required` - so never be no `docType`...

Reply:
Yeah this is a required field so it wont run. Deleting it.